src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -123,6 +179,64 @@ export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
                 </div>
               ) : null}
             </div>
+            {calculatedStats.size > 0 ? (
+              <details>
+                <summary className="arrowPanelSubTitle">
+                  Profiler Overhead
+                </summary>
+                <div className="arrowPanelSection">
+                  <div className="metaInfoGrid">
+                    <div />
+                    <div>Mean</div>
+                    <div>Max</div>
+                    <div>Min</div>
+                    {Array.from(calculatedStats).map(([key, val]) => [
+                      <div key={key}>{key}</div>,
+                      <div key={key + 'mean'}>
+                        {formatNanoseconds(val.mean)}
+                      </div>,
+                      <div key={key + 'max'}>{formatNanoseconds(val.max)}</div>,
+                      <div key={key + 'min'}>{formatNanoseconds(val.min)}</div>,
+                    ])}
```

Comment:
instead of deriving keys for each div, you can have the `map` generate fragments instead of arrays:
```js
{[...calculatedStates].map(([key, val]) => (
  <React.Fragment key={key}>
    ...
  </React.Fragment>
))}
```